src/main/java/com/purbon/kafka/topology/model/Project.java

```diff
@@ -1,7 +1,7 @@
 package com.purbon.kafka.topology.model;
 
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import com.purbon.kafka.topology.model.Impl.ProjectImpl;
+import com.purbon.kafka.topology.model.impl.ProjectImpl;
```

Comment:
I know is a technicality, but the jar is used in implementing the web-api component, so changing this to lowercase will break it. Can we probably open an issue for this and do it later when we change versions?

Reply:
Of course. Haven't found more usages than internal classes/tests. Could you point me to the web api?